fda-model/structures/Inventory.cs

```diff
@@ -260,13 +262,47 @@ public static int GetImpactAreaFID(PointM point, string polygonShapefilePath)
         }
         return -9999;
     }
+    #endregion
+
+    #region Constructors
```

Comment:
also editorial - the order of the regions we've been using is fields, properties, constructors, methods. this is not in the contributing guide, though. I will make an issue to update.